spec/features/users/log_in_spec.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+require 'rails_helper'
+
+RSpec.feature 'Authentication' do
+  it 'enforces log in to access the app' do
+    visit '/'
+    expect(page).to have_content('sign in')
+  end
+  it 'allows users to log in' do # rubocop:disable RSpec/ExampleLength
```

Comment:
What is `RSpec/ExampleLength` set to? Would it be better to extract some of this to other methods so that this passes without disabling the cop?